src/Svenkle.NuGetServer.Service/InstallerService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.ComponentModel;
+using System.Configuration.Install;
+using System.ServiceProcess;
+
+using NLog;
+
+namespace Svenkle.NuGetServer.Service
+{
+    [RunInstaller(true)]
```

Comment:
According to the documentation directly calling [ManagedInstallerClass](https://msdn.microsoft.com/en-us/library/system.configuration.install.managedinstallerclass(v=vs.110).aspx) is not intended to be used directly. Look at using [TopShelf](http://topshelf-project.com/) instead.